src/components/Footer.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Link, NavLink } from 'react-router-dom';
+// import whiteLambdaLogo from '../assets/Hire-lambda-logo-white.png';
+//we need to create and add our own logo
+
```

Comment:
Pretty sure our UI guy has logos in one of the design docs, but I could be wrong

Reply:
i fund the basic lambda ones, there isn't one specifically for lambda app store